src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/common/GetFormatDataCommand.cs

```diff
@@ -97,9 +97,15 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void ProcessRecord()
         {
-            bool writeOldWay = PowerShellVersion == null ||
-                               PowerShellVersion.Major < 5 ||
-                               (PowerShellVersion.Major == 5 && PowerShellVersion.Minor < 1);
+            // During remoting, remain compatible with older clients by default.
+            // Remoting detection: Automatic variable $PSSenderInfo is defined in true remoting contexts and also in background jobs.
+            //                     However, its Boolean .PSShowComputerName NoteProperty is true only in remoting contexts.
+            bool remoting = (GetVariableValue("PSSenderInfo") as PSObject)?.Properties["PSShowComputerName"]?.Value is bool value && value;
+            bool writeOldWay = remoting && (
+                                PowerShellVersion == null ||
+                                PowerShellVersion.Major < 5 ||
+                                (PowerShellVersion.Major == 5 && PowerShellVersion.Minor < 1)
+                                );
```

Comment:
Arbitrary indents tend to be a bit unwieldy. Do you think that perhaps something a bit more along these lines might be more readable?

```csharp
bool writeOldWay = remoting
    && (PowerShellVersion == null 
        || PowerShellVersion.Major < 5
        || (PowerShellVersion.Major == 5 && PowerShellVersion.Minor < 1));
```

Perhaps also we could rename `remoting` to something like `isRemoting` or `inRemoteSession` for additional clarity?

Reply:
I agree that the formatting should be fixed but the renaming will be reported as CodeFactor issue ""using Bulgarian prefixes".

Reply:
All good points, @vexx32 - fixed.

Reply:
@iSazonov: I went with `inRemoteSession` - will that be a problem? I've heard of _Hungarian_ prefixes - what are _Bulgarian_ ones?

Reply:
Didn't we at some point add `in` or `is` to the permitted Hungarian prefixes for codefactor? I may be misremembering. Worst case we can use `calledFromRemoteSession` but that is a bit wordy. Other suggestions also welcome, of course! 😊

Reply:
It looks like CodeFactor didn't complain about `inRemoteSession`.

Reply:
Because it is masked in Settings.StyleCop but it is not best practice.